example/src/SnackBar/SnackBarPage.tsx

```diff
@@ -1,29 +1,33 @@
 import React, { useState } from 'react';
 import { View } from 'react-native';
 import { Button, SnackBar } from 'smartway-react-native-ui';
+import type { SnackBarProps } from 'src/components/snackBar/SnackBar';
 
 export const SnackBarPage = () => {
-    const [isVisible, setVisible] = useState<boolean>(false);
+    const [snackBarProps, setSnackBar] = useState<Omit<SnackBarProps, 'setSnackBar'>>({
+        visible: false,
+        message: '',
+        actionLabel: '',
+        iconName: undefined,
+        onDismiss: undefined,
+        duration: 4000,
```

Comment:
why do we need to set this new property ? To hide the snackbar ?
I dont really see the advantage of delegate this to the caller, it makes the code more complicated in my opinion.
The component could hide the snackbar itself probably. It seems a good default behaviour for a snackbar.

Reply:
I made this, just when we for example set the snackbar, it shows, and inside snackbar components it resets the state to default. Or you mean, the snackbar message will be almost the same, we will only need to show/hide the message?

Reply:
Do we really need to reset the state of a snack bar ?
It seems easier to me to have different snack bar for different purpose.

Moreover, I have never seen a "reset" property on snackbars in other frameworks (react native paper does not provide one for example). It seems to me that if this kind of feature is desirable, it should be dealt by the user.

Reply:
I just thought it would be easier to work with it, like when we need a snackbar, we just set the state once. I used the snackbar in the revaluation task, and did this pr, to avoid onDismiss methods in the smartapp

<img width="525" alt="Screenshot 2023-03-28 at 11 51 13" src="https://user-images.githubusercontent.com/127092693/228182817-8002b4b6-3086-417f-b0c7-33b23cbad006.png">